.gitmodules

```diff
@@ -5,3 +5,4 @@
 [submodule "test/tests"]
 	path = test/jsontests
 	url = https://github.com/ethereum/tests.git
+	branch = revertTests
```

Comment:
This is not right. "branch" is only convenient feature that allows you to do `git submodule update --remote`, nothing more. We definitely don't want this to be merged into our default branch.